subworkflows/nf-core/fastq_align_dedup_bwamem/tests/gpu.nf.test

```diff
@@ -0,0 +1,224 @@
+nextflow_workflow {
+
+    name "Test Subworkflow FASTQ_ALIGN_DEDUP_BWAMEM"
+    script "../main.nf"
+    workflow "FASTQ_ALIGN_DEDUP_BWAMEM"
+    config "./nextflow.config"
+
+    tag "gpu"
+    tag "subworkflows"
+    tag "subworkflows_nfcore"
+    tag "subworkflows/fastq_align_dedup_bwamem"
+    tag "parabricks/fq2bam"
+    tag "samtools/index"
+    tag "picard/markduplicates"
+    tag "untar"
+
+    setup {
+        run("BWA_INDEX") {
+            script "../../../../modules/nf-core/bwa/index/main.nf"
+            process {
+                """
+                input[0] = Channel.of([
+                    [ id:'test' ], // meta map
+                    file('https://github.com/nf-core/test-datasets/raw/methylseq/reference/genome.fa', checkIfExists: true)
+                ])
+                """
+            }
+        }
+
+        run("BWA_INDEX", alias: 'BWA_INDEX_PE') {
+            script "../../../../modules/nf-core/bwa/index/main.nf"
+            process {
+                """
+                input[0] = Channel.of([
+                    [ id:'test' ], // meta map
+                    file('https://github.com/nf-core/test-datasets/raw/methylseq/reference/genome.fa', checkIfExists: true)
+                ])
+                """
+            }
+        }
+    }
+
+    test("Params: parabricks/fq2bam single-end | use_gpu ") {
+        when {
+            params {
+                use_gpu    = true
+                bwa_prefix = 'genome.fa'
+            }
+            workflow {
+                """
+                input[0] = Channel.of([
+                            [ id:'test', single_end:true ],
+                            file(params.modules_testdata_base_path + 'genomics/sarscov2/illumina/fastq/test_1.fastq.gz', checkIfExists: true)
+                ])
+                input[1] = Channel.of([
+                            [:],
+                            file(params.modules_testdata_base_path + 'genomics/sarscov2/genome/genome.fasta', checkIfExists: true)
+                ])
+                input[2] = Channel.of([
+                            [:],
+                            file(params.modules_testdata_base_path + 'genomics/sarscov2/genome/genome.fasta.fai', checkIfExists: true)
+                ])
+                input[3] = BWA_INDEX.out.index
+                input[4] = false // skip_deduplication

```

Comment:
Think it would be better to follow if comments actually describe then what's happening. 
`// skip_deduplication` -> `// deduplicate` if set to false and when it's `true` `// skip deduplication`

Also be consistent with comments, either provide them for all input files or none.